addons/account_analytic_analysis/account_analytic_analysis.py

```diff
@@ -655,16 +682,16 @@ def hr_to_invoice_timesheets(self, cr, uid, ids, context=None):
             'nodestroy': True,
         }
 
-    def _prepare_invoice_data(self, cr, uid, contract, context=None):
-        context = context or {}
+    @api.multi
+    def _prepare_invoice_data(self, contract):
```

Comment:
Use @api.model instead of @api.multi, where there is no operations on recordset(s) !!!